docs/hardware.md

```diff
@@ -207,18 +207,19 @@ The ROCKPro64 4 GiB board is designated as LTS (Long-term support) which means t
     </table>
 </div>
 
-## ROCK Pi
+## Radxa (ROCK Pi)
```

Comment:
Let's have only `Radxa` here, so it matches the sorter on the main website.

Reply:
Ok, done.